alg/teca_2d_component_area.h

```diff
@@ -57,18 +73,23 @@ class teca_2d_component_area : public teca_algorithm
     TECA_GET_ALGORITHM_PROPERTIES_DESCRIPTION()
     TECA_SET_ALGORITHM_PROPERTIES()
 
-    // set the name of the input array
+    // The component variable name containing the region labels.
     TECA_ALGORITHM_PROPERTY(std::string, component_variable)
 
-    // set this only if you know for certain that label ids are contiguous and
-    // start at 0. this enables use of a faster implementation.
+    /** @fn set_contiguous_component_ids
```

Comment:
use of \@fn might have been left over from a test to compare against the grouping approach?